README.md

```diff
@@ -1 +1,3 @@
-# ah-frontend-prime
+# Authors Haven - A Social platform for the creative at heart.
+
+[![Build Status](https://travis-ci.org/andela/ah-frontend-prime.svg?branch=ch-setup-travis-164798246)](https://travis-ci.org/andela/ah-frontend-prime)
```

Comment:
Link the badge to the `develop` branch instead of the `ch-setup-travis-164798246` branch.